python/lsst/analysis/tools/actions/plot/gridPlot.py

```diff
@@ -90,16 +103,49 @@ class GridPlot(PlotAction):
     def __call__(self, data: KeyedData, **kwargs) -> PlotResultType:
         """Plot data."""
         fig = plt.figure(figsize=self.figsize, dpi=self.dpi)
+        figureInfo = {"figsize": fig.get_size_inches(), "dpi": fig.get_dpi()}
+
+        if self.height_ratios is None:
+            height_ratios = np.ones(self.numRows) / self.numRows
+        else:
+            height_ratios = self.height_ratios / np.sum(self.height_ratios)
+
+        if self.width_ratios is None:
+            width_ratios = np.ones(self.numCols) / self.numCols
+        else:
+            width_ratios = self.width_ratios / np.sum(self.width_ratios)
```

Comment:
We probably need a couple of new validations down in `def validate` that verify that the length of supplied `height_ratios`/`width_ratios` (if given) matches the `numRows`/`numCols` config values.

Reply:
What happens now if you try to construct your test plot with a plot Info, but pass too many `height_ratios`? Does it fail it a meaningful error?

Reply:
Validation added.